src/test/java/com/google/devtools/build/lib/remote/GrpcRemoteCacheTest.java

```diff
@@ -656,16 +656,19 @@ public void findMissingBlobs(
               FindMissingBlobsRequest request,
               StreamObserver<FindMissingBlobsResponse> responseObserver) {
             assertThat(request.getBlobDigestsList())
-                .containsExactly(fooDigest, quxDigest, barDigest);
+                .containsAllOf(fooDigest, quxDigest, barDigest);
```

Comment:
Is it worthwhile to keep the `containsExactly` by adding the `actionKey` to the list here?

Reply:
I'd argue no, as this test checks that directory uploads work. it's a protocol detail that the action and command are also stored in the CAS that should be verified elsewhere.